linux/packaging/deb/make_config.yaml

```diff
@@ -0,0 +1,37 @@
+display_name: Feeling Finder
+package_name: feeling-finder
```

Comment:
Does the package name have to be this format, instead of the one used elsewhere
of `codes.merritt.FeelingFinder`?

Reply:
Ah, I found the doc for this. It does need to be this way apparently. :+1: